http-server/src/main/java/io/micronaut/http/server/cors/CorsUtil.java

```diff
@@ -29,13 +29,13 @@
  * @author Graeme Rocher
  * @since 1.0
  */
-class CorsUtil {
+public class CorsUtil {
```

Comment:
if we make this public annotate with `@Internal`

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/10011/commits/28b330007dbf003e6db33dfee2667faa5528ed59